recipe/meta.yaml

```diff
@@ -51,7 +50,8 @@ outputs:
         - git       # [not win]
         - m2-git    # [win]
         - ninja
-        - protobuf
+        # caffe2 doesn't compile with newer protobuf, so pinning it here
```

Comment:
If caffe2 is vendoring protobuf, do it expect to communicate with other packages using the same protobuf version? If so, I don't think this will work.

If it is only using protobuf internally, and compiling it statically, we can likely safely remove it from our dependencies.